### ISSUE TITLE
Punishment does not want to expire

### ISSUE THREAD

**Issue by Ghostiiik** [samerton/Nameless-Infractions#65]
I tried to ban the player for 1 minute. He proved active in the infraction list. But it's been almost half an hour and it's still active, even though he should have expired after one minute.

**Comment by samerton** [comment]
I'll keep this ticket open until I'm able to investigate, unless you have been able to resolve the issue @Ghostiiik?

**Comment by Ghostiiik** [comment]
No, I haven't resolved it, I don't have an overview of PHP or databases in general. Good luck, I'll use the web interface from LiteBans for now. But I'm looking forward to being able to use the module. By the way, it was me who wrote on Discord (Lukáš # 8177), so if you need to test something, for example, I'm here for you.
